vizro-core/src/vizro/static/css/aggrid.css

```diff
@@ -2,52 +2,60 @@
   --ag-background-color: var(--main-container-bg-color);
   --ag-foreground-color: var(--text-primary);
   --ag-odd-row-background-color: var(--main-container-bg-color);
+  --ag-header-foreground-color: var(
+    --text-secondary
+  ); /* Colour of text and icons in the header */
+
+  --ag-data-color: var(--text-primary); /* Colour of text in grid cells */
   --ag-header-background-color: var(--main-container-bg-color);
   --ag-header-column-resize-handle-display: none;
-  --ag-borders: none;
   --ag-icon-font-family: aggridquartz;
   --ag-icon-size: 14px;
-  --ag-row-height: 48px; /* This determines the row height, but not header */
+  --ag-row-height: 48px;
   --ag-header-height: 40px;
+
+  /* Borders */
+  --ag-borders: none;
+  --ag-row-border-style: solid;
+  --ag-row-border-color: var(--border-subtle-alpha-01);
+  --ag-row-border-width: 1px;
+
+  /* Bg Color */
+  --ag-selected-row-background-color: var(--state-overlays-selected);
+  --ag-active-color: var(--state-overlays-selected-hover);
```

Comment:
We don't really have designs for these - but I mostly took the same values as we use for the accordions